src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -294,4 +296,30 @@ static void upgradePreviewStyleFromReviewToComment(JabRefPreferences prefs) {
         String migratedStyle = currentPreviewStyle.replace("\\begin{review}<BR><BR><b>Review: </b> \\format[HTMLChars]{\\review} \\end{review}", "\\begin{comment}<BR><BR><b>Comment: </b> \\format[HTMLChars]{\\comment} \\end{comment}");
         prefs.setPreviewStyle(migratedStyle);
     }
+
+    /**
+     * The former preferences default of columns was a simple list of strings ("author;title;year;..."). Since 5.0
+     * the preferences store the type of the column too, so that the formerly hardwired columns like the graphic groups
```

Comment:
I don't see the type of the column in the preferences. Is the comment still true?

Reply:
The new form how columns are stored is divided in to parts divided by a colon. First is the column type, the second is, if existent, the qualifier. The new form is in example "groups;linked_id;field:author;field:title:special:readstatus;extrafile:chm".